identity/client/src/pages/roles/detail/RolePermissions.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from "react";
+import useTranslate from "src/utility/localization";
+import { Role } from "src/utility/api/roles";
+import EntityList from "src/components/entityList";
+import usePermissions from "src/pages/roles/modals/usePermissions";
+import { C3EmptyState } from "@camunda/camunda-composite-components";
+import { useEntityModal } from "src/components/modal";
+import EditModal from "src/pages/roles/modals/EditModal";
+import { useNavigate } from "react-router";
+
+type RoleDetailsProps = {
+  role: Role | null;
+  loading: boolean;
+};
+
+const RolePermissions: FC<RoleDetailsProps> = ({ role, loading }) => {
+  const navigate = useNavigate();
+  const { t } = useTranslate();
+
+  const { availableItems } = usePermissions();
+
+  const rolePermissions = availableItems.filter((itemX) =>
+    role?.permissions?.includes(itemX.permission),
+  );
```

Comment:
I was thinking why we can't just use `role?.permissions` and then understood that it is because `rolePermissions` is used in the entity table and contains the translated description. My idea of the `usePermissions` hook was to separate the reusable code used in both edit and create modals, so the usage here is not 100% intuitive. It would be cleaner to separate the functionality needed here (convert the plain list of string permissions to localized user facing data) to f.e. a separate hook that is also used inside `usePermissions` [here](https://github.com/camunda/camunda/blob/8cb11b8fc5b7d31c0f668483c508e8cbeec49ba6/identity/client/src/pages/roles/modals/usePermissions.tsx#L21). May be something like `useLocalizedPermissions`?